### ISSUE TITLE
`/api/draft-items?order=modified-on` should sort newest first

### ISSUE THREAD

**Issue by dpancic** [SSHOC/sshoc-marketplace-backend#370]
In GitLab by @laureD19 on Jul 22, 2022, 12:22

Sorting by last modification doesn't seem to work well in the "my draft items" section. Items should be sorted the other way around! 

![image](https://gitlab.gwdg.de/sshoc/sshoc-marketplace-frontend/uploads/35ec131b6ed661dfc6af642764829cf6/image.png)

(but ok on the "my contributed items" page)

**Comment by tparkola** [comment]
To be honest, after review of the source code, I don't think the sorting works at all. It seems like the sort parameter is not taken into consideration at all. Does it change the order for you when you select sort by label?

**Comment by stefanprobst** [comment]
you're right - i checked on https://marketplace.sshopencloud.eu/account/draft-items?order=modified-on which displays 3 items (signed in as admin) - changing the "sort by" select does not do anything

**Comment by KlausIllmayer** [comment]
Reopened because deployment did not work and I reverted the merge of the pull request, @tparkola please check this error and then again create a pull request:

Needed to revert this pullRequest due to errors in deployment: it says in src/main/java/eu/sshopencloud/marketplace/services/items/ItemsService.java:8: import eu.sshopencloud.marketplace.dto.search.SearchOrder; but such a class does not exist. Deployment thus did not work, see [action error](https://github.com/SSHOC/sshoc-marketplace-backend/actions/runs/4428655060/jobs/7768056013#step:7:32625)

You can also have a look at the pull request feedback: https://github.com/SSHOC/sshoc-marketplace-backend/pull/377 - I didn't delete the branch so it should be possible to fix the error and create a new pull request.

**Comment by tparkola** [comment]
OK, it was because of other pull requests applied before this one. Anyway, it should be fixed now, have a look at https://github.com/SSHOC/sshoc-marketplace-backend/pull/385.
